lib/modules/manager/github-actions/__fixtures__/workflow_5.yml

```diff
@@ -0,0 +1,7 @@
+jobs:
+  test:
+    runs-on: ubuntu-latest
+  next-test:
+    runs-on:
+      ubuntu-22.04
+    runs-on: "macos-12-xl"
```

Comment:
There are more valid ways to define this:

```
runs-on: 'ubuntu-22.04'
runs-on: >
  ubuntu-22.04
runs-on: |
  ubuntu-22.04
```

Is the parser automatically taking care of all the ways?